app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,11 @@
+<% provide(:title, "About") %>
+<h1>About</h1>
+<p>
+  The <a href="http://www.railstutorial.org/"><em>Ruby on Rails
+  Tutorial</em></a> is a
+  <a href="http://www.railstutorial.org/book">book</a> and
+  <a href="http://screencasts.railstutorial.org/">screencast series</a>
+  to teach web development with
+  <a href="http://rubyonrails.org/">Ruby on Rails</a>.
+  This is the sample application for the tutorial.
+</p>
```

Comment:
convention about lack of end file. You need to add new blank line at the end of file to fix